tests/testTriggers.go

```diff
@@ -684,7 +684,7 @@ func updateLongevityStats(name, eventStatName string, dashStats map[string]strin
 	name = strings.Split(name, "<br>")[0] //discarding the extra strings attached to name if any
 	version, err := Inst().V.GetDriverVersion()
 	product := "px-enterprise"
-	if eventStatName == stats.AsyncDREventName || name == stats.MetroDREventName || name == stats.StorkApplicationBackupEventName {
+	if eventStatName == stats.AsyncDREventName || eventStatName == stats.MetroDREventName || eventStatName == stats.StorkApplicationBackupEventName {
```

Comment:
https://github.com/portworx/torpedo/pull/2137#discussion_r1510020763

Reply:
Yeah Thanks, Its not pushing correctly with name.